ProjectSettings/ProjectSettings.asset

```diff
@@ -153,7 +153,7 @@ PlayerSettings:
   androidMaxAspectRatio: 2.1
   applicationIdentifier:
     Android: com.Company.ProductName
-    Standalone: Microsoft.HoloToolkit-Unity
+    Standalone: .MixedRealityToolkit
```

Comment:
We should probably leave this with the `-Unity`, to match the repo name. Was dropping the `Microsoft` intentional as well?

Reply:
Good point, I'll put the Microsoft bit back in and the suffix and then merge.